packages/harmony/src/components/layout/Flex/types.ts

```diff
@@ -10,4 +10,5 @@ export type FlexProps = {
   gap?: SpacingValue
   justifyContent?: CSSProperties['justifyContent']
   wrap?: CSSProperties['flexWrap']
+  flex?: CSSProperties['flex']
```

Comment:
This should probably exist on Box right?

Reply:
yes great call